sqlserver/datadog_checks/sqlserver/activity.py

```diff
@@ -37,50 +39,55 @@
         NVARCHAR, TODATETIMEOFFSET(CURRENT_TIMESTAMP, DATEPART(TZOFFSET, SYSDATETIMEOFFSET())), 126
     ) as now,
     CONVERT(
-        NVARCHAR, TODATETIMEOFFSET(at.transaction_begin_time, DATEPART(TZOFFSET, SYSDATETIMEOFFSET())), 126
-    ) as transaction_begin_time,
-    CONVERT(
-        NVARCHAR, TODATETIMEOFFSET(r.start_time, DATEPART(TZOFFSET, SYSDATETIMEOFFSET())), 126
+        NVARCHAR, TODATETIMEOFFSET(req.start_time, DATEPART(TZOFFSET, SYSDATETIMEOFFSET())), 126
     ) as query_start,
-    at.transaction_type,
-    at.transaction_state,
     sess.login_name as user_name,
+    sess.last_request_start_time,
```

Comment:
Should this be aliased?

```suggestion
    sess.last_request_start_time as last_request_start_time,
```

Reply:
it doesn't have to be, but probably better for consistency since I am alias-ing the others